scripts/nb-tester/qiskit_docs_notebook_tester/__init__.py

```diff
@@ -361,6 +429,104 @@ def get_args() -> argparse.Namespace:
             "The program will fail with a helpful message if any of the notebooks cannot be executed."
         ),
     )
+    parser.add_argument(
+        "--provider",
+        action="store",
+        default="qiskit-fake-provider",
+        choices=["qiskit-ibm-runtime", "qiskit-fake-provider", "runtime-fake-provider"],
+        help=(
+            "Specify a provider to run notebook against."
+        )
+    )
+    parser.add_argument(
+        "--backend",
+        action="store",
+        default="fake_athens",
+        help=(
+            "Specify a backend to run the script against, such as 'fake_athens'"
+            "or 'athens'. Only relevant when `--provider` is "
+            "`qiskit-ibm-runtime` or `runtime-fake-provider`."
+        )
+    )
+    generic_backend_options_group = parser.add_argument_group(
```

Comment:
Nit to use whitespace to make the code more readable

```suggestion
    )

    generic_backend_options_group = parser.add_argument_group(
```